Sources/CometClient/CometClient.swift

```diff
@@ -82,6 +81,11 @@ public final class CometClient {
             }
             .eraseToAnyPublisher()
     }
+
+    // TODO: implement
+    public func performAuthenticatedRequest(_ request: URLRequest) -> AnyPublisher<(data: Data, response: URLResponse), CometClientError> {
+        Empty().eraseToAnyPublisher()
```

Comment:
`fatalError()` is your friend!

Reply:
Hahaha, I have full control of this right now and nobody is using it right now, so this is fine. I will be implementing this today hopefully.